www/js/overlay.js

```diff
@@ -195,6 +201,31 @@ const toggleLarivaar = h(
   h('div.setting-label', `Use ${overlayVars.overlayLarivaar ? 'Padched' : 'Larivaar'}`),
 );
 
+const toggleCast = h(
+  'div.input-wrap.cast-toggle',
+  {
+    onclick: evt => {
```

Comment:
perhaps we should also send empty slide if toggle is being turned off? doesn't have to be this PR.